src/ImageSearchModal.jsx

```diff
@@ -65,7 +65,7 @@ export class ImageSearchModal extends React.Component {
         this.props.downloadImage(selectedImageName, this.state.imageTag, this.state.isSystem);
     }
 
-    onItemSelected(key) {
+    onItemSelected(_, key) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1266-20230413-122810-fbb171ea-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)